OpenNMT/onmt/Models.py

```diff
@@ -29,10 +29,9 @@ def __init__(self, opt, dicts):
             self.word_lut.weight.copy_(pretrained)
 
     def forward(self, input, hidden=None):
-        emb = self.word_lut(input)
-
+        batch_size = input.size(0) # batch first for multi-gpu compatibility
+        emb = self.word_lut(input).transpose(0, 1)
```

Comment:
Yeah, my bad about the previous comment. I wasn't running exactly the same thing. What I've posted above, I got from rolling back to c84f7df24bc8df556c82908a5fbd186fc38f303e and comparing with master.